_pages/dev/ssr/server-side-rendering-contextual-logging.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Server-Side Contextual Logging
+feature:
+- name: Server-Side Contextual Logging
+  spa_version: 6.2
+  cx_version: n/a
+---
+
+<TODO Verify names in code examples>
+
+Server-Side Contextual Logging is a feature that has been introduced to enhance the debugging experience for developers. With this feature, developers can see the context of the printed output and the source of the log message. Additionally, it ensures that the content is formatted correctly so that it is readable by developers, as well as monitoring tools. 
+
+In addition to the built-in solutions, this feature also comes with tools that allow customers to customize the logging experience. 
+
+Without contextual logging, the following issues arise:
+
+- Log messages are just strings with little context about the rendered URL, making it difficult to connect the log message with the source of the log.
+- Log messages are not standardized, which can make it difficult to read and understand the log messages.
+- If the log message is multiline, it is not formatted correctly, and each line is treated as a separate log in the monitoring tools, further complicating the log viewing experience.
```

Comment:
```suggestion
- If the log message is multiline, it is not formatted correctly, and each line is treated as a separate log in the monitoring tools, further complicating the log viewing experience. (This can be the case e.g. for multi-line stacktraces of errors.)
```